android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestSuggestionProvider.kt

```diff
@@ -20,13 +21,15 @@ import java.util.UUID
  * @param includeSponsoredSuggestions Whether to return suggestions for sponsored content.
  * @param includeNonSponsoredSuggestions Whether to return suggestions for web content.
  * @param suggestionsHeader An optional header title for grouping the returned suggestions.
+ * @param contextId A uuid identifier that should persist the entirety of the browser profile lifecycle.
```

Comment:
```suggestion
 * @param contextId The contextual services user identifier, used for telemetry.
```

Suggestion: Let's call out "contextual services" in this docstring, so that folks looking at this docstring for the first time can associate it with the same ID that we use for Top Sites.